bitcoin/src/blockdata/witness.rs

```diff
@@ -14,6 +17,8 @@ use crate::io::{self, Read, Write};
 use crate::prelude::*;
 use crate::VarInt;
 
+const U32_SIZE: usize = core::mem::size_of::<u32>();
```

Comment:
In 4226d602057529885cf95f935370f6e5bf68bfd2:

I think this constant hurts readability. Just use 4.